artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java

```diff
@@ -185,12 +176,7 @@ public BridgeImpl(final ServerLocatorInternal serverLocator,
       this.server = server;
    }
 
-   /** For tests mainly */
-   public boolean isBlockedOnFlowControl() {
-      return blockedOnFlowControl;
-   }
-
-   public static final byte[] getDuplicateBytes(final UUID nodeUUID, final long messageID) {
+   public static byte[] getDuplicateBytes(final UUID nodeUUID, final long messageID) {
```

Comment:
Nothing specific, just the general assumption that since `static` methods can't be overridden and since there's no apparent reason to prevent it from being hidden by a subclass then `final` is unnecessary.

Reply:
Preventing subclasses hiding statics (ick) doesnt actually seem like a bad thing which is why I wondered if there was a reason to alter the existing code when one of the comments was about adding final to other things.

Reply:
Restored `final`.